tests/sceneManager_test.cpp

```diff
@@ -10,7 +10,53 @@ using generators::PointSource;
 using objects::Object;
 using objects::TriangleObj;
 
-const double kSkipFreq = 1000;
 generators::FakeRandomGen fakeRand;
```

Comment:
Remove this as well?

Reply:
I need this to perform hitObject, so I moved it here: 
![image](https://user-images.githubusercontent.com/45213495/96715838-772e0100-13a4-11eb-9063-371bc6a69543.png)